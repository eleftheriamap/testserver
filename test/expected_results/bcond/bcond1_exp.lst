
test/exp/test/gen/bcond/bcond1_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	11259b43 	add	w3, w26, #0x966
   4:	6b03035f 	cmp	w26, w3
   8:	5400004b 	b.lt	0x10  // b.tstop
   c:	114235ce 	add	w14, w14, #0x8d, lsl #12
  10:	115f9dce 	add	w14, w14, #0x7e7, lsl #12
  14:	8a000000 	and	x0, x0, x0
