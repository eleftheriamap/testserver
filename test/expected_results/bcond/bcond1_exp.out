Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000966
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 00000000007e7000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000014
PSTATE : N---
Non-zero memory:
0x00000000 : 0x11259b43
0x00000004 : 0x6b03035f
0x00000008 : 0x5400004b
0x0000000c : 0x114235ce
0x00000010 : 0x115f9dce
0x00000014 : 0x8a000000
